<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditText</class>
 <widget class="QDialog" name="EditText">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditText</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/images/lmo.png</normaloff>:/icons/images/lmo.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbFont">
     <property name="title">
      <string>字体</string>
     </property>
     <layout class="QHBoxLayout" name="layoutFont">
      <item>
       <widget class="QCheckBox" name="chbUnderLine">
        <property name="text">
         <string>下划线</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbItalic">
        <property name="text">
         <string>斜体</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbBold">
        <property name="text">
         <string>粗体</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>颜色</string>
     </property>
     <layout class="QHBoxLayout" name="layoutColor">
      <item>
       <widget class="QRadioButton" name="rbtnRed">
        <property name="text">
         <string>红色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnGray">
        <property name="text">
         <string>灰色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnBlue">
        <property name="text">
         <string>蓝色</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="plainText">
      <string>Qt 6.5 C++
	--xyz</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutBtn">
     <item>
      <widget class="QPushButton" name="pushBtnClear">
       <property name="text">
        <string>清空</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/images/yellow.png</normaloff>:/icons/images/yellow.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>88</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushBtnOk">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/images/green.png</normaloff>:/icons/images/green.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushBtnClose">
       <property name="text">
        <string>关闭</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/icons/images/red.png</normaloff>:/icons/images/red.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushBtnOk</sender>
   <signal>clicked()</signal>
   <receiver>EditText</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushBtnClose</sender>
   <signal>clicked()</signal>
   <receiver>EditText</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>520</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
